lib/ProgramCall.js

```diff
@@ -16,25 +16,26 @@
 // DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-const iXml = require('./ixml');
-
 class ProgramCall {
   /**
    * @description creates a new ProgramCall object
    * @constructor
    * @param {string} program
    * @param {object} [options]
    */
-  constructor(program, options) {
-    if (options && typeof options === 'object') {
-      // add options if they exist, or empty string if they don't
-      this.xml = iXml.iXmlNodePgmOpen(program,
-        options.lib !== undefined ? options.lib : '',
-        options.func !== undefined ? options.func : '',
-        options.error !== undefined ? options.error : '');
-    } else {
-      this.xml = iXml.iXmlNodePgmOpen(program, '', '', '');
+  constructor(program, options = {}) {
+    this.xml = `<pgm name='${program}'`;
+
+    if (!options.error) { // set default value
+      this.xml += " error='fast'";
     }
+
+    // append options
+    if (options.lib) { this.xml += ` lib='${options.lib}'`; }
+    if (options.func) { this.xml += ` func='${options.func}'`; }
+    if (options.error) { this.xml += ` error='${options.error}'`; }
```

Comment:
Does the linter enforce the parens? I usually follow Google's style guide, which requires parens on if/else blocks _except_ when the block fits on the same line as the if.

Reply:
Parens are required but curly braces are optional when all on one line.